locust/configuration.py

```diff
@@ -58,20 +59,20 @@ def read_json(self):
                 self.config_data = json.load({})
         return self.config_data
 
-    def update_json_config(self, json_added, json_path, options, list_column, config_text):
+    def update_json_config(self, data_json, json_added, json_path, options, list_column, config_text):
         """
         Write JSON file configuration
         """
-        data = literal_eval(config_text)
+        data = data_json
 
         if(options != "replace"):
```

Comment:
there is bug with this expression. if I choose "append", the flow still not move inside this logic but run to the else logic. the `options` always contains "replace" if you print it although I choose "append"